### PR TITLE
Automatically save drafts before preview

### LINKED ISSUE #9286: Automatically save drafts before preview
When a user is editing a draft, tapping on preview should save the post and then load the preview from the site, instead of showing a fake preview.

This might require adding a loading UI to show while the post is saving.

### PR DESCRIPTION
**Pull request by yaelirub**
If a user previews a draft we automatically save it and show the site's preview instead of the fake preview.

Fixes #9286 

BEFORE:
beofre_01

AFTER:
after_01

To test:
1. Go to your site -> Blog Posts.
2. Go to Drafts.
3. Edit a draft post.
4. Tap on the 3 dots icon at the top right.
4. Tap Preview

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: WordPress/WordPress.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8218,8 +8218,8 @@
 				ORGANIZATIONNAME = WordPress;
 				TargetAttributes = {
 					1D6058900D05DD3D006BFB54 = {
-						DevelopmentTeam = PZYM8XX95Q;
 						LastSwiftMigration = 1000;
+						ProvisioningStyle = Automatic;
```
Comment (nheagy) [inline WordPress/WordPress.xcodeproj/project.pbxproj]: I don't think we can change to automatic provisioning without involving Platform9 😬

Comment (yaelirub) [reply WordPress/WordPress.xcodeproj/project.pbxproj]: mmm... oops. Not sure why it changed for me 🤔

File: WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift [hunk 2]
```diff
@@ -1186,7 +1186,17 @@ private extension AztecPostViewController {
         }
 
         alert.addDefaultActionWithTitle(MoreSheetAlert.previewTitle) { [unowned self] _ in
-            self.displayPreview()
+            if self.post.isDraft() {
```
[the remark this answers is not in the document]
Comment (yaelirub) [reply WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift]: 👍

File: WordPress/WordPress.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nheagy) [reply WordPress/WordPress.xcodeproj/project.pbxproj]: Xcode does as Xcode pleases 😆

Comment (yaelirub) [reply WordPress/WordPress.xcodeproj/project.pbxproj]: 😆

**Comment by yaelirub** [comment]
hey @nheagy any more changes needed here or can this be approved?

**Pull request merged**